src/m365/spo/commands/contenttype/contenttype-get.ts

```diff
@@ -43,7 +54,12 @@ class SpoContentTypeGetCommand extends SpoCommand {
       .get(requestOptions)
       .then((res: any): void => {
         if (res['odata.null'] === true) {
-          cb(new CommandError(`Content type with ID ${args.options.id} not found`));
+          if(args.options.id){
+            cb(new CommandError(`Content type with ID ${args.options.id} not found`));
```

Comment:
Let's avoid repetition here as well. We can define a variable named `message` above that will hold the error message and then we can call `cb(new CommandError())` once just before the `return` statement.

Reply:
done.... I guess 🤔. Please check if this is what you were thinking about? I mean the `CommandError` and `return` is done once